tests/test_csv.py

```diff
@@ -112,12 +112,12 @@ def test_checksum_zero(self):
             'bar': None,
         }
         processor.preprocess_export_row(row)
-        assert row['csum'] == '"fc43"'
+        assert row['csum'] == '@fc43'
         assert processor.validate_row(row) is None
         equiv_row = {
             'foo': '0',
             'bar': '',
-            'csum': '"fc43"'
+            'csum': '@fc43'
         }
         assert processor.validate_row(equiv_row) is None
 
```

Comment:
Have we isolated the portion of the pipeline that was causing triple-quoting to occur? Can we write a test case exercising that portion to make sure this won't similarly be quoted?

Reply:
The triple quoting comes from python csv exporting. It must be part of the standard. It turns out that quoting wasn't really the problem, but I'd still like to remove the quotes.

I added a test which does a complete export and import. 

The problem is that Excel may modify columns which shouldn't be modified.